### PR TITLE
1669 improve readme

### PR DESCRIPTION
**Pull request by JanKapala**

### CODE REVIEW

**Comment by bwilk** [comment]
looks like a nice progress

File: README.md [hunk 1]
```diff
@@ -36,31 +36,53 @@ way. To start correct elasticsearch version type `elasticsearch` in the
...
 ```
-rails dev:prime     # Remove existing resources and generate 100 new resources
-rails dev:prime[50] # Remove existing resources and generate 50 new resources
+./bin/rails dev:prime     # Remove existing resources and generate 100 new resources
+./bin/rails dev:prime[50] # Remove existing resources and generate 50 new resources
+```
+
+If this task fails when trying to seed 'services', the workaround is to run
+```shell script
+./bin/rake import:eic MP_IMPORT_EIC_URL=https://providers.eosc-portal.eu/
```
Comment (michal-szostak) [inline README.md:64]: remove `MP_IMPORT_EIC_URL=https://providers.eosc-portal.eu/` that was a workaround to the workaround :D.
The default `beta` instance should be operational.

File: README.md [hunk 2]
```diff
@@ -191,17 +223,28 @@ ENV variables:
...
+```
 
-For OSX solution might be calling `sudo launchctl config user path $PATH`
-For Linux systems modifying `PATH` in `/etc/environment` should do the job.
+Other solutions could be:
+  * For OSX: calling `sudo launchctl config user path $PATH`
+  * For Linux systems: modifying `PATH` in `/etc/environment`.
+  
+Remember that before pushing to git, `overcommit` runs rspec tests and they need running 
+[Elasticsearch server](#elasticsearch) in the background.
```
Comment (michal-szostak) [inline README.md:247]: You can add comment how to disable hooks during `commit` / `push` (run `git <command> --no-verify`

**Review by michal-szostak — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (JanKapala) [reply README.md:64]: ok xD. Is it operational now?

Comment (michal-szostak) [reply README.md]: It should be

**Review by michal-szostak — APPROVED** [review]

**Review by goreck888 — APPROVED** [review]

**Comment by goreck888** [comment]
@JanKapala Could you merge this PR?

**Pull request merged**